MagentoEcsFargate/magento_platform/magento_all.py

```diff
@@ -0,0 +1,180 @@
+from unicodedata import name
```

Comment:
I'm confused.  The previous version has everything neatly separated. Why do merge all of them in this class? We still have other stacks with same resources there. (app_stack, db_stack, es_stack, platform_stack)